frontend/src/app/modules/members/members-autocompleter.component.ts

```diff
@@ -2,8 +2,9 @@ import {UserAutocompleterComponent} from "core-app/modules/common/autocomplete/u
 import {Observable} from "rxjs";
 import {map} from "rxjs/operators";
 import {InjectField} from "core-app/helpers/angular/inject-field.decorator";
-import {HttpClient} from "@angular/common/http";
+import {HttpClient, HttpParams} from "@angular/common/http";
 import {Component} from "@angular/core";
+import {URLParamsEncoder} from "core-app/modules/hal/services/url-params-encoder";
 
 export const membersAutocompleterSelector = 'members-autocompleter';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/de19148754158145a6dce8f9e0ac392a126bccae/frontend/src/app/modules/members/members-autocompleter.component.ts#L10"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>